deleteDiscordMessages.js

```diff
@@ -224,18 +230,18 @@
             log.verb(`Estimated time remaining: ${etr}`)
             
             
-            if (myMessages.length > 0) {
+            if (messagesToDelete.length > 0) {
 
                 if (iterations < 1) {
                     log.verb(`Waiting for your confirmation...`);
                     if (!await ask(`Do you want to delete ~${total} messages?\nEstimated time: ${etr}\n\n---- Preview ----\n` +
-                        myMessages.map(m => `${m.author.username}#${m.author.discriminator}: ${m.attachments.length ? '[ATTACHMENTS]' : m.content}`).join('\n')))
+                        messagesToDelete.map(m => `${m.author.username}#${m.author.discriminator}: ${m.attachments.length ? '[ATTACHMENTS]' : m.content}`).join('\n')))
```

Comment:
I think this was supposed to be `deletableMessages`, rather than `myMessages`. Otherwise we ask "do you want to delete..." and list messages that can't be deleted.

Reply:
you're probably right, the semantics of `myMessages` seems redundant, I think would probably better to just have a `messagesToDelete` array instead and have it be the only filter to avoid confusion

Reply:
oh, now I remember why I had separate arrays for each thing, I was trying to keep track of all system messages/myMessages when the api does not correctly calculate the pagination.

if everything goes right I can stay always on the first page, as the messages keep gettin shifted to the first page, until it fills the entire first page with messages it can't delete. Anyways I have to rethink this